src/peergos/shared/user/UserContext.java

```diff
@@ -898,6 +899,17 @@ public String decryptURL(String cipherTextBase64, String nonceBase64) {
                 });
     }
 
+    public CompletableFuture<Boolean> unPublishDirectory(Path path) {
+        return getByPath(path)
```

Comment:
we want to call unShareReadAccess to ensure any friends with sub caps get sent updated caps.